app/assets/stylesheets/components/project_submissions/submissions.scss

```diff
@@ -1,6 +1,7 @@
 .submissions {
   margin-bottom: 2em;
   text-align: left;
+  width: 650px;
```

Comment:
Setting the width here is not something we can do. After growing the screen, we see this: 
![image](https://user-images.githubusercontent.com/4411701/100553211-5f6a5800-3241-11eb-8bb4-270d249c961f.png)

This will have to continue to be responsive.